src/core/math/arrayZipMap.ts

```diff
@@ -2,7 +2,16 @@ import * as R from 'ramda';
 
 type Array<T> = T[];
 
-export function arrayMap<T1, T2, TR>(
+export function arrayMap<T1, TR>(
+    a1: Array<T1>,
+    cb: (d1: T1, i: number) => TR
+) {
+    const mapArray = R.addIndex<T1, TR>(R.map);
+
+    return mapArray((iValue, i) => cb(iValue, i), a1);
+}
```

Comment:
This seems like a lot of trouble to go through to replicate `a1.map(cb)` - what's the benefit?

Reply:
None. Only benefit is keeping with a relatively consistent ramda usage.